Tests/BackendTest/Mongo2GoFixture.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Mongo2Go;
+using MongoDB.Driver;
+using Xunit;
+
+namespace BackendTest
+{
+    [CollectionDefinition("Database collection")]
+    public class DatabaseCollection : ICollectionFixture<Mongo2GoFixture>
+    {
+    }
+
+    public class Mongo2GoFixture : IDisposable
+    {
+        private const bool RUN_FOR_DEBUG = false;
+
+        private readonly MongoDbRunner _runner;
+        public  IMongoClient MongoClient { get; }
+
+        public Mongo2GoFixture()
+        {
+            _runner = RUN_FOR_DEBUG ? MongoDbRunner.StartForDebugging() : MongoDbRunner.Start();
```

Comment:
I read the HowTo manual and only thing I do not like is that I have to manually set `RUN_FOR_DEBUG = true` when I want to start debugging session. I would suggest more elegant solution using:
1. `#if DEBUG` or
2. `[Conditional("DEBUG")]` see [this](https://stackoverflow.com/a/3788719) or
3. `Debugger.IsAttached` [here](https://msdn.microsoft.com/en-us/library/system.diagnostics.debugger.isattached(v=vs.110).aspx)

Reply:
thanks, probably I have used wrong name, you can debug test, but you don't want to check data in the database...